### ISSUE TITLE
[BUG] wash fails to generate new Go/TS/Py project from template when running `wash new actor`

### ISSUE THREAD

**Issue by ericgregory** [wasmCloud/wasmCloud#1803]
### Affected project(s)

- [X] documentation
- [X] examples
- [ ] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Describe the bug

wash (0.26.0) has trouble creating a new TypeScript, Go, or Python project when running `wash new actor` and selecting a template from the menu. (Rust works fine.) Error message:

```
Failed to generate project
Invalid subfolder path: No such file or directory (os error 2)
```

It looks like the problem is that it's trying to clone the template from a subfolder that doesn't exist anymore. 

### Steps to reproduce

1. Run `wash new actor`
2. Select the Go, TypeScript, or Python templates

### Expected behavior

A new project folder generated from the template.

### Environment

- OS: MacOS
- Shell: zsh
- wash 0.26.0

### Screenshots / Logs / Additional context

<img width="837" alt="Screenshot 2024-04-02 at 8 51 07 AM" src="https://github.com/wasmCloud/wasmCloud/assets/2342090/9ba642d8-938a-487b-a341-4400e662248d">

**Comment by siddharthkhonde** [comment]
hi @ericgregory, was wondering if this is still open? happy to look into it :)

**Comment by brooksmtownsend** [comment]
This one is still open, but soon to be completely resolved with updating `wash` to work with 1.0. We could open a branch from wash 0.26 but that will likely get complicated quickly
